### PR TITLE
add declaration file

### PR DESCRIPTION
**Pull request by alexanderhodes**
Created declaration file for typescript/ expo support according to the props and the documentation from readme file
solves #13

### CODE REVIEW

**Review by RazaShehryar — APPROVED** [review]

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  /**
+   * To adjust Margin Horizontal Default: 2
+   */
+  marginHorizontal?: number;
+  // events
+  /**
+   * Trigger when scrolls and index changes
+   */
+  onScrollTo?: (index: number) => void;
+}
```
Comment (CodeBaboon) [inline index.d.ts:70]: Suggestion to add `alignDotsOnAxis`:

```suggestion
  /**
   * To adjust if the active dot should align with the x axis of the inactive dots
   * /
  alignDotsOnXAxis?: boolean;
}
```

**Comment by alexanderhodes** [comment]
@CodeBaboon Thank you for your suggestion. I've applied it and updated the PR.

**Review by CodeBaboon — APPROVED** [review]

**Review by lerouxp-abt — APPROVED** [review]

**Comment by alexanderhodes** [comment]
@yasaricli Can you merge this PR?

**Comment by IB3N** [comment]
Would really appreciate a new npm release to include this 👍

**Pull request merged**